src/tests/ftest/io/llnl_mpi4py.yaml

```diff
@@ -0,0 +1,23 @@
+hosts:
+    test_servers:
+        - boro-A
+    test_clients:
+        - boro-B
+timeout: 240
+server:
+    name: daos_server
+pool:
+    createmode:
+        mode_RW:
+             mode: 146
+    createset:
+        setname: daos_server
+    createsize:
+        size: 10000000000
+    createsvc:
+        svcn: 1
+# below mentioned path should be replaced by path of
+# LLNL and Mpi4py test suite directory in CI nodes when available.
+test_repo:
+    llnl: "/home/standan/mpiio/Testmpio"
+    mpi4py: "/home/standan/mpiio/mpi4py_latest/test"
```

Comment:
Add some more information from where to get this code. 
Should we have this repo under daos_stack or if it's not a big file may be we can maintain single file under qa repo? 
OR
We can have mpi4py as part of install and just use that single file for testing so in that case we need to add pip install mpi4py command in test document